packages/apps/docs/src/scripts/importChangelogs/utils/__tests__/prs.test.ts

```diff
@@ -0,0 +1,146 @@
+import { filterPRsWithoutData, getPrId, getPrs, getVersionPRs } from '../prs';
+
+describe('pr utils', () => {
+  describe('getPrId', () => {
+    it('should return the prID from the content with at start and label cleaned', () => {
+      const content = 'Support for hyperlane-message-id (#1335)';
+      const result = getPrId(content);
+
+      expect(result.label).toEqual('Support for hyperlane-message-id');
+      expect(result.prIds[0].tries).toEqual(0);
+      expect(result.prIds[0].id).toEqual(1335);
+      expect(result.commits.length).toEqual(0);
+    });
+
+    it('should return the multiple commits between the brackets', () => {
+      const content = 'Support for verifier plugins (#1324,#1336)';
+      const result = getPrId(content);
+
+      expect(result.label).toEqual('Support for verifier plugins');
+      expect(result.prIds.length).toEqual(2);
+      expect(result.prIds[0].id).toEqual(1324);
+      expect(result.prIds[1].id).toEqual(1336);
+      expect(result.commits.length).toEqual(0);
+    });
+
+    it('should return the multiple commits between the brackets', () => {
+      const content =
+        'Fixed issue with the hash of cap guards, hash native and principals (#1273) (#1278) (#1287)';
+      const result = getPrId(content);
+
+      expect(result.label).toEqual(
+        'Fixed issue with the hash of cap guards, hash native and principals',
+      );
+      expect(result.prIds.length).toEqual(3);
+      expect(result.prIds[0].id).toEqual(1273);
+      expect(result.prIds[1].id).toEqual(1278);
+      expect(result.prIds[2].id).toEqual(1287);
+      expect(result.commits.length).toEqual(0);
+    });
+
+    it('should return when there is no commit found', () => {
+      const content = 'I have the powerrrrr';
+      const result = getPrId(content);
+
+      expect(result.label).toEqual('I have the powerrrrr');
+      expect(result.commits.length).toEqual(0);
+    });
+  });
+
+  describe('getPrs', () => {
+    it('should return an array with all commits', async () => {
+      const { default: changelog } = await import('./mock/changelog.json', {
+        assert: {
+          type: 'json',
+        },
+      });
+
+      const pkg = changelog['Pact 4'] as unknown as IChangelogPackage;
+
+      const result = getPrs(pkg);
+      expect(result.length).toEqual(185);
+    });
+    it('should return an array of commits', async () => {
+      const { default: changelog } = await import('./mock/changelog.json', {
+        assert: {
+          type: 'json',
+        },
+      });
+
+      const pkg = changelog['Pact 4'] as unknown as IChangelogPackage;
+      const result = getPrs(pkg);
+
+      expect(result[0].id).toEqual(1326);
+    });
+  });
+
+  describe('getVersionPRs', () => {
+    it('should return an array with all commits', async () => {
```

Comment:
```suggestion
    it('should return an array with all commits', async () => {
```
this could be better wording to describe what it does return because even the above test cases `getPrs` also have the same test describes but the return data is different, so it makes sense to add that context. 

even for below test scenarios